Link zu angular.schule/blog/2022-11-use-define-for-class-fields (#19)
angular-buch/website-articles: blog/2022-11-use-define-for-class-fields/README.md

* [Propertys zukunftssicher initialisieren](/blog/2022-11-use-define-for-class-fields#propertys-zukunftssicher-initialisieren)
* [Auswirkungen auf bestehenden Angular-Code](/blog/2022-11-use-define-for-class-fields#auswirkungen-auf-bestehenden-angular-code)

> > **🇬🇧 Dieser Artikel ist auch in englischer Sprache verfügbar: [TypeScript: useDefineForClassFields – How to avoid future Breaking Changes](https://angular.schule/blog/2022-11-use-define-for-class-fields)**

Reviewer: ```suggestion
> **🇬🇧 This article is available in English language here: [TypeScript: useDefineForClassFields – How to avoid future Breaking Changes](https://angular.schule/blog/2022-11-use-define-for-class-fields)**
```

> > **🇬🇧 Dieser Artikel ist auch in englischer Sprache verfügbar: [TypeScript: useDefineForClassFields – How to avoid future Breaking Changes](https://angular.schule/blog/2022-11-use-define-for-class-fields)**

Author: Stimmt!
